config/environments/development.rb

```diff
@@ -16,16 +16,20 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
+  # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
-  config.action_mailer.default_url_options = { host: 'dev.fsektionen.se' }
+  config.action_mailer.default_url_options = { host: 'fsektionen.se' }
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.perform_deliveries = true
-  config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.default charset: 'utf-8'
+  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.default :charset => "utf-8"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.